wordpress-mobile/WordPress-iOS
# [Post Preview] Adds NewPostPreviewViewController for redesign work

Fixes #13180 (once `postPreview` feature flag is disabled)

Part of #13197

This is intended to be a rough copy of the existing class but will change as additional functionality is added (such as Desktop/Mobile previews)

To test:

- Preview a post from Edit, Post List, or other screen
- Check that new view controller is used

PR submission checklist:

- [x] ~I have considered adding unit tests where possible.~ I think the `NewPostPreview` will undergo some changes and tests should be written once it is complete.

- [x] ~I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.~ This can be done once the feature flag is released.
